### DISCUSSION TITLE
Tweak some ambiguous wording

### DISCUSSION THREAD

**Discussion by ShrimpTacos314** [mdn/mdn-community#545] [Content]
So, recently I was looking along the button docs in the hopes of reminding myself how to run JavaScript when I pressed a button. I scrolled down to the "Attributes" heading, and I saw a line of text which read "This element's attributes include the global attributes."

I took this to mean that the global attributes were in the list. Obviously they were not, and I began to think that maybe `onClick` didn't exist or, worse, it was left out (_gasp!_). I wound up opening an issue, where I was then informed about the problem.

My point is this: if the global attributes are not listed under the "Attributes" heading, then that should be made clearer. Maybe change the phrase to something like this:

"This list does not include the global attributes."
"The global attributes also apply to this element."

That way, new users on the site or people who don't know English quite as well don't accidentally misunderstand.

**Comment by Ryuno-Ki** [comment]
https://github.com/mdn/content/issues/7460 is the issue in question

**Comment by pepelsbey** [comment]
Closing as answered/settled.
